client/src/libs/validate.js

```diff
@@ -0,0 +1,14 @@
+export const isTouched = (touchedState) => {
+  return !!Object.keys(touchedState).length;
```

Comment:
I think this will return true, if the input's text is deleted, once it is entered.

Reply:
I think, once the user enters the value and then delete it and exits the input filed, then also it has to display the error saying it is a required field.

Reply:
Please add the test for this.